src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-size: center;
+  background-image: url('../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+
+  &:nth-child(-n + 4) {
```

Comment:
The selector '.stars__star:nth-child(-n + 4)' will apply the active star image to the first 4 stars only, regardless of the rating value. This should be dynamically set based on a class applied to the '.stars' block that indicates the current rating (e.g., 'stars--4').